static/app/data/experimentConfig.tsx

```diff
@@ -25,6 +25,12 @@ export const experimentList = [
     parameter: 'exposed',
     assignments: [0, 1],
   },
+  {
+    key: 'DefaultBusinessForAllExperiment',
```

Comment:
@Neo-Zhixing I'd probably call this experiment something like `DefaultBusinessForFreeOrgsExperiment` or something that denotes it's only for free orgs.

Reply:
lol naming is hard. name changed.